src/pages/ServiceList/ServiceListComponent.tsx

```diff
@@ -130,7 +138,7 @@ class ServiceListComponent extends React.Component<ServiceListComponentProps, Se
     });
   }
 
-  updateServices() {
+  updateServices(rateInterval?: string) {
```

Comment:
Maybe a default `rateInterval`?

Reply:
Sure! Apparently, typescript allows to do `updateServices(rateInterval = MetricsOptionsBar.DefaultRateInterval)` in the definition of the function. I'll use that.